apps/docs/content/components/html-elements/A.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: A
+description: A specialized box component for HTML anchor element.
```

Comment:
it mentions "box component", but it's not the case anymore. it's an orbiter sentence